Makefile.COMMON

```diff
@@ -61,21 +65,18 @@ SELFLINK ?= $(GOPATH)/src/$(ROOTPKG)
 
 default: $(BINARY)
 
-.build/$(GOPKG):
+$(GOCC):
+	@echo Go version $(GO_VERSION) required but not found in PATH.
+	@echo Abort now if you want to manually install it system-wide instead.
```

Comment:
It can seem a bit confusing when running it because it's not clear that what it's downloading will not be system-wide (might scare some people who are not aware of how permissions work or who are building as root).

How about adding this line in the middle:

    @echo Go version $(GO_VERSION) required but not found in PATH.
    @echo Installing Go $(GO_VERSION) into self-contained local build environment.
    @echo Abort now if you want to manually install it system-wide instead.

(or something like that)